templates/controller-deployment.yaml

```diff
@@ -52,11 +52,27 @@ spec:
       {{- end }}
       {{- end }}
       containers:
-      - args:
-        - controller
-        - --enable-leader-election
-        command:
-        - consul-k8s
+      - command:
```

Comment:
Changed to the way we do this everywhere else.

Reply:
heh..there is merge conflict waiting to happen here with the `webhook` PR but it should be trivial